auth-web/src/util/constants.ts

```diff
@@ -165,14 +165,40 @@ export enum LearFilingTypes {
 // see also https://github.com/bcgov/bcrs-shared-components/blob/main/src/modules/corp-type-module/corp-type-module.ts
 export enum CorpTypes {
     // actual corp types
+
+    // in colin but modernized
     BC_CCC = 'CC',
     BC_COMPANY = 'BC',
     BC_ULC_COMPANY = 'ULC',
+
+    // modernized
```

Comment:
Good comments but the codes will become "modernized" via a FF, so I recommend not having these comments after all.

Reply:
Resolved in #2433 .